crates/claims/crates/vc/src/syntax/mod.rs

```diff
@@ -102,7 +102,7 @@ pub struct TypedObject {
     pub extra_properties: BTreeMap<String, json_syntax::Value>,
 }
 
-pub(crate) mod value_or_array {
+pub mod value_or_array {
```

Comment:
Can you make it back as private if you didn't use it?

Reply:
oh, sorry! I thought I did-- let me fix it!